### DISCUSSION TITLE
Edit Heating and DHW Programs

### DISCUSSION THREAD

**Discussion by Vippis2000** [emsesp/EMS-ESP32#2076] [Q&A]
Is it possible to edit the time schedules for heating and warm water through EMS-ESP?

Until now its only possible via KM200 API in IOBroker.

Furthermore is it possible to have more than 2 time setpoints for the heating schedule?

For warm water it is possible to enter more than 2 setpoints.

Background: I want to make a 24h optimization for HP operation which takes PV forecast and dynamic electricity price into account. So i would like to enter a 24h schedule to the HP every day.

**Comment by MichaelDvP** [comment]
#1594
